### DISCUSSION TITLE
Why did you pick `druid` as the UI toolkit for this project?

### DISCUSSION THREAD

**Discussion by edfloreshz** [jpochyla/psst#262] [Q&A]
I was wondering what made you chose `druid` as the UI toolkit for this project.
- Did you already had some experience with the toolkit?
- What were your considerations?
- Why not other toolkits like `egui` or `iced`?

@jpochyla

**Comment by jpochyla** [comment]
Hi!

I didn't have any experience with Druid before starting Psst. Druid was, not counting some web-based experiments, the second library I've tried. The first was Iced, and I've created a very small prototype in it. Comparing to Druid, the UI development in Iced is MUCH easier, but I was attracted to a certain professionalism I've seen in the people behind Druid: focus on the hard-but-necessary things like IME, text layout, accessibility, and a real desire to put a good desktop UI framework on the table. That  said, the development of Druid is not really progressing much, and I think that some of its issues, keeping it from reaching that goal, require a lot of consistent thought and a BDFL rather than community-style development. But I still think it has the best shot, at least based on the code and knowledge that's already there.
